package.json

```diff
@@ -4,11 +4,12 @@
   "license": "MIT",
   "description": "",
   "scripts": {
-    "test": "jest -c test/jest.config.js",
+    "test": "jest",
     "start": "start-storybook -p 6006",
     "set-storybook-title": "node ./scripts/set-document-title.js HyperTwind",
     "build-storybook": "build-storybook && yarn set-storybook-title",
-    "create-component": "node ./scripts/create-component.js"
+    "create-component": "node ./scripts/create-component.js",
+    "repl": "node -r @swc-node/register -r ./scripts/init-repl.ts"
```

Comment:
Use `@swc-node/register` for faster repl.

https://user-images.githubusercontent.com/7823011/188505096-2fc90a6e-2798-4990-89a9-24216ee15e6f.mov